Bylaws.mediawiki

```diff
@@ -70,8 +70,19 @@
 ## ensuring that they are subscribed to the members-announce email list.
 ## ensuring they have signed a copy of the current version of the Hacklab waiver.
 ## paying dues as set by the board.
-
-# Members may resign by written notice to memberadmin@hacklab.to
+## leaving the space as clean as or better than the member finds it after each visit.
+## reporting damage or problems with the equipment or space to the area steward, or the Board if no steward is available, as soon as possible.
```

Comment:
Should "don't break the anti-harassment policy" be added to member responsibilities?

https://github.com/hacklabto/hacklab-bylaws/blob/7fb44162ac3b9b424134997d64e11c5c55a3147c/Anti-Harassment_Policy.mediawiki